### ISSUE TITLE
All the drop area text resizes twice when you drag a file over it

### ISSUE THREAD

**Issue by Softvision-CristinaBadescu** [mozilla/send#658]
[Affected versions]:
- Firefox 57.0.1 and up

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Steps to reproduce]:
1. Navigate to the https://send.stage.mozaws.net/
2. Drag a file over the Drop area and observe the text size.

[Expected result]:
- The text is bigger.

[Actual result]:
- The text is smaller, then it becomes bigger.

[Notes]:
- On Linux, this issue isn't as visible as on the other mentioned OS's, but there is a slight change in the font size.
- This issue is also reproducible on https://send.firefox.com/.
- This issue is not reproducible on Chrome.
- Here is a screen recording of the issue:
![text size](https://user-images.githubusercontent.com/31244235/33717034-1fbca698-db61-11e7-9bae-4e7eb9260866.gif)

**Comment by dannycoates** [comment]
🤷‍♂️ It's scaling with a css transform, but I don't see anything weird with it. I imagine this is a browser painting / font rendering issue. @ericawright any suggestions?

**Comment by ericawright** [comment]
I played with some numbers with our scaling, it seems like it does not happen if we scale greater than ~1.2, but it happens if it is less than that.
With safari - it is roughly the same behavior, but the scale is around 1.7 before it stops happening to the smaller italic test.
I also tried it on Chrome, and the issue is not there, so I believe it is also a browser issue, something with how we scale text?
(Chrome does have another issue, where the centered text appears to move to the right by ~1 pixel, and changing our css didn't seem to fix that either.)
